README.md

```diff
@@ -1 +1,12 @@
-TBD
+# Taskmaster [![42](https://i.imgur.com/9NXfcit.jpg)](i.imgur.com/9NXfcit.jpg)
+
+Taskmaster is a 42 project. It is an implementation of a **process supervisor** and aims at getting familiar with job control mechanisms on UNIX-like operating systems.
+
+---
+
+### Description
+
```

Comment:
If we're not putting anything here, I think we should write something like TBD or in progress :)